### ISSUE TITLE
sigmoid vs ReLu

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#71]
sigmoid = logistic activation function 

sigmoid:
Therefore, it is especially used for models where we have to predict the probability as an output.Since probability of anything exists only between the range of 0 and 1, sigmoid is the right choice.

The logistic sigmoid function can cause a neural network to get stuck at the training time.
The softmax function is a more generalized logistic activation function which is used for *multiclass classification*

![](https://miro.medium.com/max/1400/1*XxxiA0jJvPrHEJHD4z893g.png)
.
